### ISSUE TITLE
Sister Module

### ISSUE THREAD

**Issue by smbambling** [boundedinfinity/bi-puppet-banner#1]
This really isn't a issue, but I wanted to let you know that this module was the inspiration for https://github.com/arineng/file_header.  The major difference is that we do not require the ripienaar/puppet-module-data dependency module.

**Comment by boundedinfinity** [comment]
@smbambling Cool.  I'll check out the other project.  I'm still a unsure if using the ripienaar/puppet-module-data was a "best practice" or not.  After some testing at work, it seems that it doesn't work very well with some older puppet masters.  In fact I'm not convinced that heira is worth the trouble in general.

I may be reworking the it to remove the module data stuff.

Anyway, I'm glad you found it useful.  Thanks for the note.

**Comment by smbambling** [comment]
Yeah module-data is a really neat idea...but seems like its still young.  Your more then welcome to fork what we have it doesn't require hiera but can leverage it if needed.
